data/remnant/remnant missions.txt

```diff
@@ -4487,3 +4487,120 @@ mission "Remnant: Expanded Horizons Astral job"
+			choice
+				`	"Are there any special adjustments needed?"`
+				`	"The <ship> is ready to go."`
+					goto readyforstorms
+			`	"Well, I expect you are already setup to handle the normal Ember Waste heat, correct?" She continues without waiting for you to reply. "The Ember Waste storms primarily are a risk to electrical systems - typically by draining your power. So you can compensate by ensuring that you have some excess available to use. I know some captains carry extra generators to ensure they always have extra power, but others instead carry extra crystal capacitors so they can outlast the storm. In theory, you could just accept that the your ship will be shut down during the height of the storm, and reboot everything afterwards. So long as no threats are present, it is safe but somewhat nervewracking. Which strategy fits the <ship> is up to you."`
+			label readyforstorms
+			`	"I will settle into a bunk to do some pre-calculations. Please let me know when we arrive in Aescolanus."`
+				accept
+	npc kill
+		government "Test Dummy"
+		system "Aescolanus"
+		ship "_Ion Timer Ship" "Timer"
+		personality unconstrained heroic nemesis
+		dialog `As the storm starts to swell around you the instrumention chimes to indicate that it has collected all the data it needs. Dawn quickly begins the shutdown routine to protect the equipment against potential damage, and gives you the OK to return to <destination>.`
+	on complete
+		payment 250000
+		conversation
+			`"Thank you for your help, <first>. Sometimes I have to wait a while before we have a ship available to do missions like these." She pauses, then taps her commlink. "Speaking of which, I have confirmed the transfer of the logistics fee of <payment> to your account."`
+			choice
+				`	"I am always happy to help. What are you working on now, anyway?"`
+				`	"I am always happy to help. See you around!"`
+					goto stormend
+			`	"How familiar are you with the nature of space?"`
+			choice
+				`	"Not very, to be honest."`
+				`	"I know space is a vacuum, and can be either extremely hot or extremely cold depending on circumstances..."`
+					goto basics
+			`	She nods "Honesty is good. It is hard to learn when we are not willing to admit that we do not know something."`
+				goto explanation
+			label basics
+			`	She nods "Those are the basics, a good starting point."`
+			label explanation
+			`	A hologram of the Ember Waste appears over her commlink. "As you have no doubt picked up on, the Ember Waste do not conform to humanity's commonly understood rules about space. For instance, the standard background temperature is 2.B1 degrees, and the average density of the interstellar medium is F4240 particles per cubic meter. Here in the Ember Waste, the average background temperature is an order of magnitude greater and has an average density of E8D4A51000 particles per cubic meter."`
```

Comment:
```suggestion
			`	A hologram of the Ember Waste appears over her commlink. "As you have no doubt picked up on, the Ember Waste does not conform to humanity's commonly understood rules about space. For instance, the standard background temperature is 2.B1 degrees, and the average density of the interstellar medium is F4240 particles per cubic meter. Here in the Ember Waste, the average background temperature is an order of magnitude greater and has an average density of E8D4A51000 particles per cubic meter."`
```

Also I feel like these confusing numbers would be better expressed in a more comprehensible format. I get that she's brainy and explaining something technical, but we don't want the player to feel stupid here.

You might also consider moving the option to ask about the temperature system here, when it's first encountered.